NCU-Project/Model/Users/Post.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Post.swift
+//  NCU-Project
+//
+//  Created by Simon Rowlands on 19/10/2018.
+//  Copyright © 2018 simonrowlands. All rights reserved.
+//
+
+import Foundation
+
+struct Post {
+    let id: Int
+    let userID: Int
+    let title: String
+    let body: String
+    
+}
+
+extension Post: Decodable {
+    
+    enum PostKeys: String, CodingKey {
```

Comment:
could this be renamed to just Keys so that it is called Post.Keys instead of Post.PostKeys?